cordova-lib/src/platforms/platforms.js

```diff
@@ -102,5 +102,10 @@ function getPlatformProject(platform, platformRootDir) {
 }
 
 module.exports = platforms;
-module.exports.getPlatformProject = getPlatformProject;
-module.exports.PlatformProjectAdapter = PlatformProjectAdapter;
+
```

Comment:
Ok found it, adding `writable: true` to `defineProperties` solves the test failures. Pushing. Thanks for the pull request.